code/modules/uplink/uplink_items.dm

```diff
@@ -509,11 +509,11 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	include_modes = list(/datum/game_mode/nuclear)
 	exclude_modes = list(/datum/game_mode/nuclear/clown_ops)
 
-/datum/uplink_item/dangerous/bolt_action
-	name = "Surplus Rifle"
-	desc = "A horribly outdated bolt action weapon. You've got to be desperate to use this."
-	item = /obj/item/gun/ballistic/rifle/boltaction
-	cost = 1
+/datum/uplink_item/dangerous/surplus_smg
+	name = "Surplus SMG"
+	desc = "A horribly outdated automatic weapon. Why would you want to use this."
+	item = /obj/item/gun/ballistic/automatic/plastikov
+	cost = 3
```

Comment:
why does something thats worse than a plasma cutter in practically every way cost 3 tc

Reply:
like there is literally no reason to ever use this just use a stechkin